### PR TITLE
editors/vscode: Add syntax highlighting for `extern c`

### PR DESCRIPTION
**Pull request by Lenart12**

### CODE REVIEW

File: editors/vscode/syntaxes/jakt.tmLanguage.json [hunk 1]
```diff
@@ -784,8 +784,16 @@
           "match": "\\b(as)\\b\\s"
         },
         {
-          "name": "keyword.control.other.jakt",
-          "match": "\\b(import)\\b"
+          "name": "keyword.control.import.jakt",
+          "match": "\\b(import)(?:\\s*)(?:(?<=\\s+)(?:(extern\\s+c)|(extern)))?\\b",
+          "captures": {
+            "2": {
+              "name": "keyword.control.import.extern-c.jakt"
```
Comment (alimpfard) [inline editors/vscode/syntaxes/jakt.tmLanguage.json:791]: :thinking: that's a valid scope name? I thought they had to match `(\w+\.)+\w+`

File: editors/vscode/syntaxes/jakt.tmLanguage.json [hunk 2]
```diff
@@ -784,8 +784,16 @@
           "match": "\\b(as)\\b\\s"
         },
         {
-          "name": "keyword.control.other.jakt",
-          "match": "\\b(import)\\b"
+          "name": "keyword.control.import.jakt",
+          "match": "\\b(import)(?:\\s*)(?:(?<=\\s+)(?:(extern\\s+c)|(extern)))?\\b",
```
Comment (alimpfard) [inline editors/vscode/syntaxes/jakt.tmLanguage.json:788]: Why the unbounded backwards assertion? that's technically not valid in ecma262
```
\b(import)(?:\s*)(?:(\bextern\s+c)|(\bextern))?\b
```
sounds just fine

File: editors/vscode/syntaxes/jakt.tmLanguage.json (hunk 1 above)
Comment (Lenart12) [reply editors/vscode/syntaxes/jakt.tmLanguage.json:791]: There is already scopes like `fat-arrow` and `return-throws`. If that is true we should fix all of them

Comment (alimpfard) [reply editors/vscode/syntaxes/jakt.tmLanguage.json:791]: Oh, if it works then it's alright, it's just a name

File: editors/vscode/syntaxes/jakt.tmLanguage.json (hunk 2 above)
Comment (Lenart12) [reply editors/vscode/syntaxes/jakt.tmLanguage.json]: Changed to your propsal :)

**Pull request merged**